src/user.ts

```diff
@@ -1088,14 +1094,16 @@ export class User {
         email: newEmail,
         token: URLSafeUUID()
       };
-      await this.mailer.sendEmail(
-        'confirmEmailChange',
-        user.unverifiedEmail.email,
-        {
-          req: req,
-          user: user
-        }
-      );
+      if (!this.config.mailer.useCustomMailer) {
```

Comment:
Here is no event emited

Reply:
The event is emitted here:

https://github.com/perfood/couch-auth/blob/325148330d65f4950ba46ec08b637e07eff25f4d/src/user.ts#L1116